package-lock.json

```diff
@@ -0,0 +1,4400 @@
+{
```

Comment:
Did you write all of this?

Reply:
package.json and package-lock.json are automatically generated by npm

Reply:
The goat as appeared, I need to practice reviewing code but everything here looks good so far as I see it